developing_packages/editable_packages.rst

```diff
@@ -248,6 +248,12 @@ time, without any Conan command.
 
     When a package is in editable mode, most of the commands will not work. It is not possible to :command:`conan upload`,
     :command:`conan export` or :command:`conan create` when a package is in editable mode.
+    
+.. note::
+
+   CMake-based consumer projects may not pick up on modifications made to header files in the editable package. In this 
```

Comment:
I think this behavior is not specific to CMake, but any build system and working with editable packages

Reply:
I mean, this is the normal (maybe not expected at first try) behavior

Reply:
That's entirely fair, it's just that I offered to add a note in the documentation in the linked Conan issue. 

I think it's enough of a potential trip-up that I figured it could save people troubleshooting time if one would point this out up-front. But feel free to close the PR if you don't think that's necessary.

Reply:
I think the note is fine, just make it more generic (it doesn't happen only with header libraries but any other library as well) and is not specific to CMake